modules/tinymce/src/plugins/help/main/ts/data/PluginUrls.ts

```diff
@@ -61,6 +61,7 @@ const urls = Arr.map<PartialPluginUrl, PluginUrl>([
   { key: 'mediaembed', name: 'Enhanced Media Embed', type: PluginType.Premium, slug: 'introduction-to-mediaembed' },
   { key: 'export', name: 'Export', type: PluginType.Premium },
   { key: 'formatpainter', name: 'Format Painter', type: PluginType.Premium },
+  { key: 'inlinecss', name: 'InlineCss', type: PluginType.Premium },
```

Comment:
I can't remember what we decided last time. Do we need a changelog entry for this? It is a user-facing change.

Reply:
Not sure. Couldn't find any changelog entry about adding new plugin to the list

Reply:
I think this suggests it shouldn't have one: 

https://github.com/tinymce/tinymce/pull/8072#discussion_r953315580

Approving.

Reply:
Is this name correct? Should it be this instead based on the Jiras?
```suggestion
  { key: 'inlinecss', name: 'Inline CSS', type: PluginType.Premium },
```

Reply:
I should add it looks like the DoD for TINY-9328 is incorrect/inconsistent, as looking at everything else (e.g. the Epic) it is `Inline CSS` not `InlineCss`.

Reply:
I agree with Lee

- [ ] Correct the display name